Bottom: 51e443ade0a4f88b19ed694f2e8b1c5d9eaf1440
Top:    6c1e4cf53330ee6255062b2aa3a77c802dd15072
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2010-05-31 10:22:27 +0200

remove FormatCompiler tests


---

diff --git a/objectformatter/src/test/scala/net/virtualvoid/string/EvaluateSpecs.scala b/objectformatter/src/test/scala/net/virtualvoid/string/EvaluateSpecs.scala
index f49d047..d9100cc 100644
--- a/objectformatter/src/test/scala/net/virtualvoid/string/EvaluateSpecs.scala
+++ b/objectformatter/src/test/scala/net/virtualvoid/string/EvaluateSpecs.scala
@@ -56,22 +56,6 @@ object EvaluateSpecs extends Specification{
   "The format interpreter" should {
     evaluate(ObjectFormatter)
   }
-  "The format compiler" should {
-    evaluate(FormatCompiler)
-  }
-}
-
-object FormatAccountsExample{
-  def main(args:Array[String]){
-    System.out.println("Hello")
-    System.out.println(FormatCompiler.format(
-"""#name has these bank accounts:
-  #accs[#number at #bank.name having these transactions:
-    #transactions[#isWithdrawal?[Withdrawal|Deposit]: #amount Euros at #date->date[dd.MM.yyyy]]{ 
-    }*]{
-  }*
-""",EvaluateSpecs.thePerson))
-  }
 }
 
 class EvaluateSpecsTest extends runner.JUnit4(EvaluateSpecs)
